client/src/components/Tool/ToolCardSection.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+const props = defineProps({
+    title: String,
+});
+</script>
+
+<template>
+    <div class="mt-2 mb-4">
+        <div v-if="title" class="title-bar my-2">
+            <div class="stripe"></div>
+            <h2>{{ props.title }}</h2>
+            <div class="stripe"></div>
+        </div>
+
+        <div v-else class="stripe"></div>
+        <slot />
+    </div>
+</template>
+
+<style scoped lang="scss">
+@import "scss/theme/blue.scss";
+
+.title-bar {
```

Comment:
Looking at this here. I think it is ok to leave the tool section title change in this PR. The component is well written but we should remove some/most of these custom styles if possible. Ideally, custom styles should rarely contain explicit sizes.

Reply:
I can make the sizes dependent on the global theme where possible, but removing the styles is not possible, without changing the design.

Reply:
I think using h4 this way would be an accessibility issue: https://www.w3.org/WAI/tutorials/page-structure/headings/

Reply:
Interesting, then it has to be handled separately, since it is already an accessibility issue also for the current tool, history and workflow panels and unrelated to this PR.